### ISSUE TITLE
Windows 11/ARM64: INTERNAL-ERROR-VkQueue-state-timeout

### ISSUE THREAD

**Issue by corporateshark** [KhronosGroup/Vulkan-ValidationLayers#8961]
**Environment:**
 - OS: Windows 11/ARM 64
 - GPU and driver version: Qualcomm(R) Adreno(TM) X1-45 GPU; API version 1.3.276.0
 - SDK or header version if building from repo: Vulkan SDK 1.3.296 for ARM64
 - Options enabled (synchronization, best practices, etc.): standard Vulkan Configurator, GPA/Descriptor-Indexing

**Describe the Issue**

1) Run [Chapter04/Ch04_Sample06_DemoApp](https://github.com/PacktPublishing/3D-Graphics-Rendering-Cookbook-Second-Edition/blob/main/Chapter04/06_DemoApp/src/main.cpp) (or any other example from that repo).

2) Press `Esc` to close the application.

3) Notice that everything freezes upon exit, followed by this error message after a long delay.

```
INTERNAL-ERROR-VkQueue-state-timeout(ERROR / SPEC): msgNum: 1331811241 - Validation Error: [ INTERNAL-ERROR-VkQueue-state-timeout ] Object 0: handle = 0x20a0c7fe780, type = VK_OBJECT_TYPE_QUEUE; | MessageID = 0x4f61d3a9 | vkDeviceWaitIdle():  The Validation Layers hit a timeout waiting for queue state to update (this is most likely a validation bug). seq=5967 until=5967
    Objects: 1
        [0] 0x20a0c7fe780, type: 4, name: NULL

ERROR:
Validation layer:
 Validation Error: INTERNAL-ERROR-VkQueue-state-timeout
 Object 0: handle = 0000020A0C7FE780, type = VK_OBJECT_TYPE_QUEUE;
 MessageID = 000000004F61D3A9
 vkDeviceWaitIdle():  The Validation Layers hit a timeout waiting for queue state to update (this is most likely a validation bug). seq=5967 until=5967
```

**Expected behavior**

Clean exit.

**Comment by spencer-lunarg** [comment]
@corporateshark is this something you are finding only specific to Windows ARM devices?

**Comment by corporateshark** [comment]
> @corporateshark is this something you are finding only specific to Windows ARM devices?

Yes, I've just got LVK and the demos running on Windows 11/ARM64 with Snapdragon, but I ran into this deadlock in Validation Layers.

**Comment by spencer-lunarg** [comment]
this is unfortunate, but currently no one on the Validation Layers team have a Windows ARM device (we have one internally in our CI) ... basically it might be a while before someone will be able to get a device to look into this

**Comment by artem-lunarg** [comment]
I'm interested in Windows ARM specifics. Will try to get the device. Not sure if can get to this till the end of the year, because of other tasks though.

**Comment by artem-lunarg** [comment]
@corporateshark It turns out that a fix we did here https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/8766 also fixes the behavior from this ticket. Available in the latest code and will be available in the next SDK.
